test/clj/game_test/engine/rules.clj

```diff
@@ -660,3 +660,136 @@
         (core/play-dynamic-ability state :runner {:dynamic "auto-pump" :card (refresh ank)})
         (is (= 3 (:current-strength (refresh ank))) "Ankusa is at 3 strength")
         (is (= 1 (:credit (get-runner))) "Spent 3 to pump")))))
+
+(deftest autoresolve
+  (testing "Aeneas with and without autoresolve"
+    (do-game
+     (new-game {:corp {:deck ["Jackson Howard"]}
+                :runner {:deck [(qty "Aeneas Informant" 2)]}})
+     (play-from-hand state :corp "Jackson Howard" "New remote")
+     (take-credits state :corp)
+     (core/gain state :runner :click 50)
+     (play-from-hand state :runner "Aeneas Informant")
+     (letfn [(run-jackson []
+               (run-empty-server state "Server 1")
+               (click-prompt state :runner "No action"))
+             (get-aeneas1 [] (get-resource state 0))]
```

Comment:
Just so you know, this is effectively the same as `(let [aeneas (get-resource state 0)]` and then using `(refresh aeneas)` everywhere you wrote `(get-aeneas1)`. I don't think it's worth changing your tests over, but it is good to be aware of.

Reply:
Thanks for the reminder. This was on purpose, but only done out of laziness - I keep getting annoyed at forgetting to refresh cards, so saving the, uh, 4 characters seemed worthwhile. I won't really defend it, though. :P